src/libraries/System.Private.CoreLib/src/System/Reflection/ConstructorInvoker.cs

```diff
@@ -56,6 +93,11 @@ public object Invoke()
             return InvokeImpl(null, null, null, null);
         }
 
+        /// <inheritdoc cref="Invoke(object?)"/>
```

Comment:
NIT: It looks to me like referencing itself, shouldn't be the parameter less invoke?
```suggestion
        /// <inheritdoc cref="Invoke()"/>
```

Reply:
Ah the build already failed for parameters mismatch
`error CS1573: (NETCORE_ENGINEERING_TELEMETRY=Build) Parameter 'constructor' has no matching param tag in the XML comment for 'ConstructorInvoker.Create(ConstructorInfo)' (but other parameters do)`

Reply:
Thanks. Doesn't appear inheritdoc has validation, and also apparently doesn't support the argument list (have to duplicate).